addons/website/views/snippets/s_timeline.xml

```diff
@@ -2,51 +2,60 @@
-                <div class="s_timeline_date"><b>2019</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <h5 class="card-title">Your title</h5>
-                            <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
-                        </div>
-                    </div>
-                    <i class="fa fa-1x fa-child bg-o-color-2 rounded-circle s_timeline_icon"/>
+    <section class="s_timeline pt48 pb48" data-vcss="001">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col-lg-12 pb24 text-center" data-name="Heading">
+                    <h2 class="h3-fs">Latest news</h2>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
                 </div>
-                <div class="s_timeline_content"/>
             </div>
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2018</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">You can edit, duplicate...</p>
+            <div class="position-relative pt-3">
+                <div class="s_timeline_row position-relative d-flex gap-5 align-items-start pb-4" data-name="Milestone">
+                    <div class="s_timeline_line position-absolute top-0 bottom-0 d-block w-0 mb-1 border-start pe-none"/>
+                    <span class="s_timeline_dot o_not_editable position-absolute translate-middle-x d-flex rounded-circle pe-none text-o-color-1" contenteditable="false"/>
+                    <div class="s_timeline_content w-100 ps-4 ps-md-0">
+                        <div class="s_timeline_card s_card card my-0 ms-auto text-md-end" style="border-width: 0px !important;" data-vcss="001" data-vxml="001" data-snippet="s_card" data-name="Milestone Event">
+                            <div class="card-body">
+                                <small class="text-muted">13/06/2019</small>
+                                <h3 class="h4-fs card-title">First Feature</h3>
+                                <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
+                            <a href="#" class="btn btn-primary">See more</a>
+                            </div>
                         </div>
                     </div>
-                    <i class="fa fa-1x fa-graduation-cap bg-o-color-2 rounded-circle s_timeline_icon"/>
+                    <div class="s_timeline_content w-100"/>
```

Comment:
> ... but the issue with that is that it then keeps the alignment set on desktop view and so we cannot change it 🤔

This was because of the `gap-5` defined on its parent. I fixed the issue by removing the gap for mobile 👍 .